lld/test/ELF/aarch64-thunk-bti.s

```diff
@@ -0,0 +1,513 @@
+// CHECK-EXE4-NEXT:           60 00 00 18       .word   0x18000060
+// CHECK-EXE4-NEXT:           00 00 00 00       .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT:  <__AArch64AbsLongThunk_>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x30000074 <__AArch64AbsLongThunk_+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           68 00 00 18       .word   0x18000068
+// CHECK-EXE4-NEXT:           00 00 00 00       .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT: <__AArch64AbsLongThunk_>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x30000084 <__AArch64AbsLongThunk_+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           70 00 00 18        .word   0x18000070
+// CHECK-EXE4-NEXT:           00 00 00 00        .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT: <__AArch64AbsLongThunk_>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x30000094 <__AArch64AbsLongThunk_+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           78 00 00 18        .word   0x18000078
+// CHECK-EXE4-NEXT:           00 00 00 00        .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT: <__AArch64AbsLongThunk_>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x300000a4 <__AArch64AbsLongThunk_+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           80 00 00 18        .word   0x18000080
+// CHECK-EXE4-NEXT:           00 00 00 00        .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT: <__AArch64AbsLongThunk_>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x300000b4 <__AArch64AbsLongThunk_+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           90 00 00 18        .word   0x18000090
+// CHECK-EXE4-NEXT:           00 00 00 00        .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT: <__AArch64AbsLongThunk_>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x300000c4 <__AArch64AbsLongThunk_+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           98 00 00 18        .word   0x18000098
+// CHECK-EXE4-NEXT:           00 00 00 00        .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT: <__AArch64AbsLongThunk_via_plt>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x300000d4 <__AArch64AbsLongThunk_via_plt+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           d0 00 00 18        .word   0x180000d0
+// CHECK-EXE4-NEXT:           00 00 00 00        .word   0x00000000
+// CHECK-EXE4-EMPTY:
+// CHECK-EXE4-NEXT: <__AArch64AbsLongThunk_absolute>:
+// CHECK-EXE4-NEXT:           ldr     x16, 0x300000e4 <__AArch64AbsLongThunk_absolute+0x8>
+// CHECK-EXE4-NEXT:           br      x16
+// CHECK-EXE4-NEXT:           00 00 00 f0        .word   0xf0000000
+// CHECK-EXE4-NEXT:           00 00 00 00        .word   0x00000000
+
+//--- lds
+PHDRS {
+  low PT_LOAD FLAGS(0x1 | 0x4);
+  high PT_LOAD FLAGS(0x1 | 0x4);
+}
+SECTIONS {
+    .text_low 0x10000000 : {
```

Comment:
Unintended 4-column when `PHDRS` uses 2-column indentation?

Reply:
ACK. I've made the indentation consistent.